clients/tests/rocsparse_test_main.cpp

```diff
@@ -173,6 +174,74 @@ class ConfigurableEventListener : public testing::TestEventListener
 
 int main(int argc, char** argv)
 {
+    //
+    // Enable debug mode for testing.
+    //
+    rocsparse_enable_debug();
+
+    //
+    // Enable test debug arguments.
+    //
+    if(rocsparse_clients_envariables::is_defined(
+           rocsparse_clients_envariables::TEST_DEBUG_ARGUMENTS)
+       == false)
+    {
+        rocsparse_clients_envariables::set(rocsparse_clients_envariables::TEST_DEBUG_ARGUMENTS,
+                                           true);
+    }
+
+    //
+    // Disable debug argument verbose
+    //
+    {
+        static constexpr const char* option_force_debug_arguments_verbose
```

Comment:
Should this be moved to the rocsparse_parse_data file?

Reply:
Yes, it can.

Reply:
Done.